rust/Makefile.am

```diff
@@ -61,6 +61,7 @@ CARGO_ENV +=	LOCALSTATEDIR=$(e_localstatedir)
 all-local: Cargo.toml
 	mkdir -p $(abs_top_builddir)/rust/gen
 	cd $(abs_top_srcdir)/rust && \
+		CC="$(CC)" CFLAGS="$(CFLAGS)" \
```

Comment:
Does this really make a change ?

Reply:
Yes. Without it for example, suricata-lua-sys was always using "cc" to compile.

Reply:
Wondering why it fails only in one build then...

Reply:
What about `CFLAGS="$(CFLAGS) -Wno-error"` ? (maybe unless in rust strict mode)

Reply:
> Wondering why it fails only in one build then...

This is the only job that uses `-Wimplicit-int-conversion`, and that's where the error lies.

Reply:
Argh we need to add `-Wimplicit-int-conversion` to `--enable-warnings` default options

Reply:
(Oh I guess this is because of unit tests using quick and dirty code...)